src/test/kotlin/no/nav/syfo/application/api/PrediksjonApiSpek.kt

```diff
@@ -142,23 +149,50 @@ class PrediksjonApiSpek : Spek({
                 }
 
                 it("Should return BadRequest if no fnr is given as header") {
+                    val counter = COUNT_PREDIKSJON_OUTPUT_FAILED.get()
                     with(
                         handleRequest(HttpMethod.Get, url) {
                             addHeader(Authorization, bearerHeader(validToken))
                         }
                     ) {
                         response.status() shouldBeEqualTo HttpStatusCode.BadRequest
+                        COUNT_PREDIKSJON_OUTPUT_FAILED.get() shouldBeEqualTo counter.inc()
                     }
                 }
 
                 it("Should return 403 Forbidden if veileder doesn't have access to user") {
+                    val counter = COUNT_PREDIKSJON_OUTPUT_FORBIDDEN.get()
                     with(
                         handleRequest(HttpMethod.Get, url) {
                             addHeader(Authorization, bearerHeader(validToken))
                             addHeader(NAV_PERSONIDENT_HEADER, ARBEIDSTAKER_VEILEDER_NO_ACCESS.value)
                         }
                     ) {
                         response.status() shouldBeEqualTo HttpStatusCode.Forbidden
+                        COUNT_PREDIKSJON_OUTPUT_FORBIDDEN.get() shouldBeEqualTo counter.inc()
+                    }
+                }
+            }
+            describe("Database down") {
+                it("should return error when database fails") {
+                    database.stop()
```

Comment:
Mottakelig for ideer om en bedre måte å gjøre dette på.

Reply:
Det går kanskje an å legge til en egen start-metode i `TestDB`, så slipper man å lage en ny når man vil starte igjen?

Reply:
Tror jeg fant en bedre måte å gjøre det på.